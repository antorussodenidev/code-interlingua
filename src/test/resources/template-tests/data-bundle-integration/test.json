{
  "nm": {
    "nodeType": "Expression",
    "code": "k.a[0].x"
  },
  "j": 42,
  "k": { "a": [{ "x": "k.a[0].x" }] }
}
