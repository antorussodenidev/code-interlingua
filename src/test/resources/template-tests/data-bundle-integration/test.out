class C {
  C() {
    super("k.a[0].x", k.a[0].x);
  }
}
